extensions/asyncapi/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "AsyncAPI",
+  "title": "AsyncAPI",
+  "description": "AsyncAPI extension for Raycast brings some AsyncAPI tools to you.",
+  "icon": "command-icon.png",
+  "author": "Sergio Moya <smoya89@gmail.com> (sergiomoya.me)",
```

Comment:
This should just be your Raycast handle / username.

Reply:
Yeah, and it is so clear in the guidelines! Sorry, this is fixed now!